packages/rx_bloc_cli/lib/src/commands/create_command.dart

```diff
@@ -59,6 +60,11 @@ class CreateCommand extends Command<int> {
         help: 'Enables Firebase analytics for the project',
         allowed: ['true', 'false'],
         defaultsTo: 'false',
+      )
+      ..addOption(
+        _facebookAuthString,
+        help: 'Enables facebook authentication for the project',
+        defaultsTo: 'true',
```

Comment:
Optional features should be disabled by defauult.

Reply:
We agreed that social logins should be enabled by default

Reply:
Please have a look at this PR - https://github.com/Prime-Holding/rx_bloc/pull/464